packages/docs/docs/agent/index.md

```diff
@@ -173,6 +173,12 @@ npm run agent <base_url> <client_id> <client_secret> <agent_id>
 
 Keep the terminal open so that you can see the logs. At this point you can start sending messages to the agent.
 
+## Supported Protocols
+
+- [HL7 v2](https://www.hl7.org): A low-level transfer protocol for HL7 content.
+- [DICOM C-Store](https://dicom.nema.org/dicom/2013/output/chtml/part07/sect_9.3.html): A protocol used for storing metadata about medical images.
```

Comment:
"DICOM" is the protocol

"C-STORE" (all caps) is one operation

We support both "C-STORE" and "C-ECHO" today.

If we wanted to put a salesy spin, we could say "other DICOM operations such as C-FIND, C-GET, and C-MOVE are coming soon"

Reply:
@ksmith94 can you update to clarify the distinction between the protocol and the operations?